util/src/main/scala/geotrellis/util/Filesystem.scala

```diff
@@ -53,6 +53,24 @@ object Filesystem {
   }
 
   /**
+    * Read the contents of a file into a ByteBuffer.
+    *
+    * @param   path The path to the file to be read
+    * @return       A ByteBuffer that contains the file
+    */
+  def streamByteBuffer(path: String): ByteBuffer = {
```

Comment:
I think "stream" here is not the right nomenclature. What exactly is this byte buffer that is being returned? what is the type of `channel`...is this really a file map buffer? Let's make sure to correctly name it.